### ISSUE TITLE
Results with AdjLogits

### ISSUE THREAD

**Issue by anohdnu** [hyunjunChhoi/Balanced_Energy#2]
Hi Thank you for your great work. How can I reproduce the results with AdjLogits for CIFAR-LT ACC?

**Comment by hyunjunChhoi** [comment]
> Hi Thank you for your great work. How can I reproduce the results with AdjLogits for CIFAR-LT ACC?

I use the same strategy of 

stage2.py in this repo  https://github.com/amazon-science/long-tailed-ood-detection 

Based on our final checkpoint, we train adj-logits as same as stage2  tuning methodology

**Comment by anohdnu** [comment]
Thank you very much.
How can I calculate the prior_gamma1, and prior_gamma2 values for a new dataset like ImageNet-LT dataset?
